packages/client/src/utils.ts

```diff
@@ -2,3 +2,11 @@ import range from "lodash/range";
 
 export const repeat = (length: number, ret: string) =>
   range(length).map(() => ret);
+
+export const fieldsErrorsMapping: { [key: string]: string } = {
```

Comment:
As typescript is structurally typed, you can remove this. 
This way, if you try to access an unknown key, you will get an error!
```suggestion
export const fieldsErrorsMapping = {
```

Reply:
I think if I want to use a string as an index for an object I need to precise the key is a string in the type of this object

Reply:
Otherwise I get the error:
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ "required": string; "wrongEmail": string; "wrongPassword": string; "differentPassword": string; "passwordTooShort": string; }'.
  No index signature with a parameter of type 'string' was found on type '{ "required": string; "wrongEmail": string; "wrongPassword": string; "differentPassword": string; "passwordTooShort": string; }'